docs/deploy/subnet_discovery.md

```diff
@@ -23,12 +23,13 @@ Private subnets are used for internal load balancers. These subnets must have th
 
 
 ## Common tag
-In version v2.1.1 and older, both the public and private subnets must be tagged with the cluster name as follows:
```

Comment:
Lets keep the first two statements
- In version v2.1.1 and older ...
- The cluster tags is not required in v2.1.2...

and add the following
- Starting v2.4.2 release, you can disable the cluster tag check completely by specifying the feature gate `SubnetsClusterTagCheck=false`